index.html

```diff
@@ -17,6 +17,7 @@
   ga('create', 'UA-89848325-1', 'auto');ga('send', 'pageview');
   </script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">

```

Comment:
It feels like a little overkill to add Bootstrap for just one class. Can we just use the CSS of "row" and add it to our stylesheet instead?